packages/react-ui/tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+	entry: ['source/index.tsx'],
+	format: ['cjs', 'esm', 'iife'],
```

Comment:
I think "iife" is not needed (wasn't there before, no user asked it, obsolete format)